src/AppBundle/Resources/config/doctrine/Cuisine.orm.yml

```diff
@@ -20,4 +20,8 @@ AppBundle\Entity\Cuisine:
             unique: false
             nullable: true
             precision: 0
+    manyToMany:
+      owners:
```

Comment:
This generates the following error:
```
Property AppBundle\Entity\Cuisine::$owners does not exist
```
It should be `restaurants`, I suppose, but you would need to add the corresponding property to the `Cuisine` class.
Just to be sure: why did you do this? To be able to lookup all restaurants matching a certain cuisine?

Reply:
Yes - restaurants can have several cuisines